cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -91,16 +92,35 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
   });
 
   it('deactivating a kontainer driver should hide its card from the cluster creation page', () => {
+    cy.intercept('GET', '/v3/kontainerdrivers').as('getKontainerDrivers');
+    cy.intercept('POST', 'v3/kontainerDrivers/azurekubernetesservice?action=deactivate').as('deactivateDriver');
+    cy.intercept('POST', 'v3/kontainerDrivers/azurekubernetesservice?action=activate').as('activateDriver');
+
     const driversPage = new KontainerDriversPagePo();
     const clusterCreatePage = new ClusterManagerCreatePagePo();
 
     // deactivate the AKS driver
     KontainerDriversPagePo.navTo();
     driversPage.waitForPage();
+
+    // assert kontainer driver is in Active state
+    cy.wait('@getKontainerDrivers').then(({ response }) => {
+      response.body.data.forEach((item: any) => {
+        if (item.id === 'azurekubernetesservice') {
+          const state = item['active'];
```

Comment:
Please, observe that in the` body.data` of this object, there's another attribute `status:active `. Can you please, verify if there's any difference in how this data is being consumed that could affect the test execution?

Reply:
@IsaSih could you elaborate the cases that might impact the execution here and why is it relevant for the scenario?

Reply:
Me and @yonasberhe23 verified that both attributes change when the element changes its state so it seems like any of them can be used. The API payloads are just steps to ensure that the state of the element being tested changed, so that we then set `reenableAKS` accordingly